core/modules/startup/rootwidget.js

```diff
@@ -26,7 +26,7 @@ exports.startup = function() {
 		$tw.modal.display(event.param,{variables: event.paramObject, event: event});
 	});
 	$tw.rootWidget.addEventListener("tm-show-switcher",function(event) {
-		$tw.modal.display("$:/core/ui/SwitcherModal",{variables: event.paramObject, event: event});
+		$tw.modal.display("$:/core/ui/SwitcherModal",{variables: event.paramObject, event: event, maskClosable: true});
```

Comment:
I think `maskCloseable` is already part of the `paramObject` @Jermolene .. right?
So it doesn't need to be added in the root-widget, where it is hardcoded. IMO it needs to be configurable